classes/class-sensei-media-attachments.php

```diff
@@ -224,6 +224,16 @@ public function display_attached_media() {
 			__( 'Lesson Media', 'sensei_media_attachments' ) :
 			__( 'Course Media', 'sensei_media_attachments' );
 
+		/**
+		 * Change the media heading title on course and lesson pages.
+		 *
+		 * @since 2.0.0
+		 *
+		 * @param string  $media_heading Heading text for course or lesson page's attachments.
+		 * @param int     $post_id       Current post ID.
+		 */
+		$media_heading = apply_filters( 'sensei_media_attachments_media_heading', $media_heading, $post->ID );
```

Comment:
What are your thoughts on supplying the `$post_type` as another argument? I included it in the filter above (see https://github.com/woocommerce/sensei-media-attachments/pull/42). Not strictly necessary, but seems like it would be handy and save a step.

If you think we should not include it, then for consistency we should change the above filter as well.

Reply:
I had it in before, but then thought they could do everything with the `$post_id`. They technically don't need that because this is always in the loop. I'll put `$post_type` back in for now.